ProjFS.Mac/PrjFSLib/PrjFSKextLogDaemon/org.vfsforgit.prjfs.PrjFSKextLogDaemon.plist

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Label</key>
+	<string>org.vfsforgit.prjfs.PrjFSKextLogDaemon</string>
+	<key>Program</key>
+	<string>/usr/local/libexec/PrjFSKextLogDaemon</string>
```

Comment:
I'll fix this up in a future PR where we'll install this as part of VFSForGit.

Reply:
Great, thanks!